tests/data/test_callbacks.py

```diff
@@ -147,8 +144,9 @@ def configure_data_fetcher(*args, **kwargs) -> CustomBaseVisualization:
 
     for stage in _STAGES_PREFIX.values():
 
-        for _ in range(10):
-            getattr(dm, f"show_{stage}_batch")(reset=False)
+        for fcn_name in _PREPROCESS_FUNCS:
```

Comment:
Keep the previous one, it was asserting we could iterate more than the dataset length and the iterator was being reset.

Reply:
And let's add yours too.